automation/project/Settings.scala

```diff
@@ -41,7 +41,7 @@ object Settings {
   val commonSettings =
     commonBuildSettings ++ testSettings ++ List(
     organization  := "org.broadinstitute.dsde.firecloud",
-    scalaVersion  := "2.11.8",
+    scalaVersion  := "2.12.4",
```

Comment:
Why not 2.12.6 (the latest)?

Reply:
I found out that Scalatest 3.0.5 is compatible with Scala 2.12.4. There isn't any compile error with Scala 2.12.6. However, not gaining anything with 2.12.6 in UI automation.
https://oss.sonatype.org/content/groups/public/org/scalatest/scalatest_2.12/3.0.5/scalatest_2.12-3.0.5.pom